lib/arena.rb

```diff
@@ -1,2 +1,49 @@
 class Arena
+
+  attr_accessor :name, :gladiators
+
+  def initialize(name)
+    @name = name.capitalize
+    @gladiators = []
+  end
+
+  def add_gladiator(gladiator)
+    @gladiators << gladiator
+    # puts "adding gladiator #{gladiator}"
+  end
+
+  def add_gladiator(gladiator)
+    if @gladiators.length < 2
+      @gladiators << gladiator
+    end
+  end
+
+  def fight # no gladiators, does nothing
+  end
+
+  def start_fight(weapon)
+    if @gladiators.length == 2
+      #compare weapons (tridentVspear, clubVspear, tridentVclub, sameVsame)
+
+      #create if statements with weapon combinations
+      if @gladiators.weapon(0) && @gladiators.weapon(1)
+        # remove gladiator with spear from array
+        @gladiators.delete_at(0)
+      #end
+
+    #   elsif @gladiators.weapon(0) && @gladiators.weapon(2)
```

Comment:
Good job realizing that this code shouldn't be there.  Get the first one working, then repeat the working concept.